common/common.cpp

```diff
@@ -647,9 +681,15 @@ void gpt_print_usage(int /*argc*/, char ** argv, const gpt_params & params) {
     printf("  --cfg-negative-prompt-file FNAME\n");
     printf("                        negative prompt file to use for guidance. (default: empty)\n");
     printf("  --cfg-scale N         strength of guidance (default: %f, 1.0 = disable)\n", params.cfg_scale);
-    printf("  --rope-scale N        RoPE context linear scaling factor, inverse of --rope-freq-scale\n");
+    printf("  --rope-scaling {none,linear,yarn}\n");
+    printf("                        RoPE frequency scaling method, defaults to linear unless specified by the model\n");
+    printf("  --rope-scale N        RoPE context scaling factor, inverse of --rope-freq-scale\n");
```

Comment:
I came across this a while back, but just saying it is the "inverse" of another option, which also says it is the "inverse" of this option gives you effectively no information.
Something like "larger than 1.0 to elongate rope for larger context sizes" or something at one of the 2 options would be gold.